about.html

```diff
@@ -80,7 +80,7 @@ <h1 class="about-title">About</h1>
       <a href="https://www.facebook.com/darya.tolok.750" class="fa fa-facebook"></a>
       </div>
       <p>
-        Photo: Anton Tolok & Tim Tolok
+        Photo: Anton Tolok, Tim Tolok, Elizabeth Popova
```

Comment:
I would suggest for the last person use `&` instead of `,`, what do you think?

Reply:
Yes, that is a great catch.